backend/tracim_backend/models/data.py

```diff
@@ -686,11 +687,17 @@ def new_from(cls, revision: 'ContentRevisionRO') -> 'ContentRevisionRO':
 
         new_rev.updated = datetime.utcnow()
         if revision.depot_file:
-            new_rev.depot_file = FileIntent(
-                revision.depot_file.file.read(),
-                revision.file_name,
-                revision.file_mimetype,
-            )
+            try:
+                new_rev.depot_file = FileIntent(
+                    revision.depot_file.file.read(),
+                    revision.file_name,
+                    revision.file_mimetype,
+                )
+            except IOError as exc:
+                raise NewRevisionAbortedDepotCorrupted(
+                    "Can't create new revision of content"
+                    " if last revision file is not accessible."
```

Comment:
Some remark. You can suggest to check that the file is available, but do not say "the file is not available" - you did not check it)

Reply:
done